app/src/main/java/com/mapbox/navigation/examples/fetchroute/FetchARouteActivity.kt

```diff
@@ -96,6 +101,19 @@ class FetchARouteActivity : AppCompatActivity() {
         binding.fetchARouteButton.setOnClickListener { fetchARoute() }
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        mapboxNavigation.onDestroy()
+    }
```

Comment:
Remove mapboxNavigation.onDestroy calls, `MapboxNavigationApp` is responsible for calling destroy